src/components/common/Pill/index.tsx

```diff
@@ -6,15 +6,23 @@ const Pill = ({
   variant,
   textStyleClasses,
   containerClasses,
+  widthFull = false,
 }: PillProps) => {
   let backgroundColor;
+  let textColor;
```

Comment:
don't use mutiple vars for each function. simply use classnames and apply classes based on variants